web_src/js/features/common-global.js

```diff
@@ -142,16 +142,12 @@ export function initGlobalCommon() {
     }
   });
 
-  // loading-button this logic used to prevent push one form more than one time
-  $(document).on('click', '.button.loading-button', function (e) {
```

Comment:
Could this break `<a>`s with the `button` style class?
Links can't be submitted, right?
But is there even a link that satisfies alll these requirements?
I haven't found any with the `loading-button` class, but simple `button` links are common...
(Of course, that was only a small search I did...)

Reply:
Yes, `<a>` can not submit a form in HTML, only through JS would that be possible. Searching for `loading-button` in `templates` yields exactly 3 buttons that use this class, so I think we should keep the selector simple and independent of the Fomantic-specific `.button` class.